src/browser/convert_options.js

```diff
@@ -239,6 +240,8 @@ module.exports = {
             plugins: newOptions.plugins
         };
 
+        newOptions.webPreferences.disableInitialReload = newOptions.experimental.disableInitialReload;
```

Comment:
Move this into https://github.com/HadoukenIO/core/pull/277/files#diff-bb76f534ea36dcd8884c45b25ab0f7b0R238